//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// check_feasibility.h
//
// Code generation for function 'check_feasibility'
//

#ifndef CHECK_FEASIBILITY_H
#define CHECK_FEASIBILITY_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
unsigned char check_feasibility(double V_wayp, double A_wayp, double V_max,
                                double V_min, double A_max, double A_min,
                                double J_max, double J_min);

#endif
// End of code generation (check_feasibility.h)
